generate_documentation_qmd/action.yml

```diff
@@ -0,0 +1,95 @@
+name: 'Build and publish qmd'
+author: 'Dries Schaumont'
+description: >
+  Create quarto markdown files for viash components or workflows, based on a jinja template.
+inputs:
+  input_dir:
+    required: false
+    description: 'Directory where the viash configs can be found.'
+    default: "."
+  component_template:
+    required: false
+    description: >
+      Jinja template that can be used to customize the resulting qmd files.
+      By default will use the included template.
+  output_dir:
+    required: true
+    description: >
+      Output folder to write to.
+  dest_path:
+    required: false
+    description: Template for output paths
+    default: "{type}s/{namespace}/{name}.qmd"
+  git_tag:
+    required: false
+    description: >
+      The git tag. Used to create links to the source code in the generated documentation.
+    default: "${{ github.ref_name }}"
```

Comment:
Will this default actually work? If not, could do something like:

```bash
git_tag="${{ inputs.git_tag }}"
if [ -z "$git_tag" ]; then
  git_tag="${{ github.ref_name }}"
fi
```

Reply:
This works.